### PR TITLE
feat: Gatsby: schema.org: return of lost changes in schemas

### PR DESCRIPTION
**Pull request by redoper1**
Return of some changes lost by error when merging "master" in PR #351 and then reverting it in #353

I hope that now it will merge as expected 🙂

### CODE REVIEW

File: gatsby/src/templates/situations/page.tsx [hunk 1]
```diff
@@ -37,10 +37,16 @@ const Page: React.FC<IProps> = ({ data, pageContext }) => {
         htmlLanguage={pageContext.langCode}
       />
       <SchemaComp
-        datePublished={data.situation.valid_from}
+        datePublished={
+          data.situation.valid_from
+            ? data.situation.valid_from
```
Comment (rostaklein) [inline gatsby/src/templates/situations/page.tsx:42]: Should we rather preffer `creatted`? 🤔

File: gatsby/src/components/schema/schema.tsx [hunk 2]
```diff
@@ -96,6 +106,21 @@ export const SchemaComp: React.FC<IProps> = ({
...
+      faqList.push({
+        '@type': 'Question',
+        name: questions_answer.question,
+        acceptedAnswer: {
+          '@type': 'Answer',
+          text: questions_answer.value,
+        },
+      });
+    });
+  }
```
Comment (rostaklein) [inline gatsby/src/components/schema/schema.tsx:122]:
```suggestion
  const faqList = questions_answers?.map((questions_answer) => ({
  '@type': 'Question',
  name: questions_answer.question,
  acceptedAnswer: {
    '@type': 'Answer',
    text: questions_answer.value,
  },
}));
```
Would probably just work as well? :) and its much cleaner.

**Review by rostaklein — COMMENTED** [review]
Any way to check if the schema is valid or so? 🤔 (other than checking the page source code)

File: gatsby/src/templates/situations/page.tsx (hunk 1 above)
Comment (redoper1) [reply gatsby/src/templates/situations/page.tsx:42]: I`m not sure 🤔

**Comment by redoper1** [comment]
> Any way to check if the schema is valid or so? 🤔 (other than checking the page source code)

There isn't any other way as far as I know.

File: gatsby/src/components/schema/schema.tsx (hunk 2 above)
Comment (redoper1) [reply gatsby/src/components/schema/schema.tsx:122]: Yeah, it works too and it is a lot cleaner, you are right.

File: gatsby/src/templates/situations/page.tsx (hunk 1 above)
Comment (redoper1) [reply gatsby/src/templates/situations/page.tsx:42]: Reason for this form is that some of the situations have the value of `valid_from` set to a date before it was created. But I don't know if it's somehow relevant that validity for this structured data or there should be the real `created` date.

**Review by vitkolos — APPROVED** [review]

File: gatsby/src/templates/situations/page.tsx (hunk 1 above)
Comment (redoper1) [reply gatsby/src/templates/situations/page.tsx:42]: i think we can leave it like this for now and if it will be needed we can edit it later.

**Pull request merged**